compiler/src/iree/compiler/Dialect/LinalgExt/Transforms/ConvertConv2DToWinograd.cpp

```diff
@@ -463,12 +493,14 @@ struct ConvertConv2DToWinogradPass
   void runOnOperation() override {
     MLIRContext *context = &getContext();
     RewritePatternSet patterns(&getContext());
+    auto moduleOp = getOperation();
+    SymbolTable symbolTable(moduleOp);
     patterns.insert<FoldWinogradFilterTransform<linalg::Conv2DNchwFchwOp>,
-                    FoldWinogradFilterTransform<linalg::Conv2DNhwcHwcfOp>,
-                    ConvertConvToWinograd<linalg::Conv2DNhwcHwcfOp>,
+                    FoldWinogradFilterTransform<linalg::Conv2DNchwFchwOp>>(
```

Comment:
This looks like a repeat. We should be running this before the conversion from nchw -> nhwc convolutions.